lib/ruby_lsp/addon.rb

```diff
@@ -1,6 +1,8 @@
 # typed: strict
 # frozen_string_literal: true
 
+require_relative "response_builder"
```

Comment:
This will be necessary for most of the LSP, so I think we can move this require to `ruby_lsp/internal` where we require most parts.